### DISCUSSION TITLE
Error: relation "collection_groups_collection_id_group_id" does not exist when updating version from 0.71

### DISCUSSION THREAD

**Discussion by milotiger** [outline/outline#6960] [Self Hosting]
When trying to update from v0.71.0 to v0.76.2; I got the migration error, please see below
```
[api] info: [database] Running migrations... 
[api] info: [database] Migrating 20230921071140-rename-collection-groups.js... 
[api] Migration 20230921071140-rename-collection-groups.js (up) failed: Original error: relation "collection_groups_collection_id_group_id" does not exist
```

**To Reproduce**
Update from v0.71.0 to v0.76.2

**Expected behavior**
The database will be migrated successfully and the app will be updated

**Screenshots**
<img width="720" alt="image" src="https://github.com/outline/outline/assets/16278766/e1a214e7-44a3-43a0-9af5-e5d1d3d8f01f">

**Outline (please complete the following information):**
- Install: [self hosted]
- Version: [Update from v0.71.0 to v0.76.2]

**Comment by linear** [comment]
<p><a href="https://linear.app/outline/issue/OLN-394/error-relation-collection-groups-collection-id-group-id-does-not-exist">OLN-394 Error: relation "collection_groups_collection_id_group_id" does not exist when updating version from 0.71</a></p>

**Comment by tommoor** [comment]
I'm probably not going to be able to fix anything here as this version is over 6 months old, I'd recommend looking at your database relationships and renaming it so that it does exist. Perhaps the migration failed part way?

**Comment by milotiger** [comment]
> I'm probably not going to be able to fix anything here as this version is over 6 months old, I'd recommend looking at your database relationships and renaming it so that it does exist. Perhaps the migration failed part way?

Yes, I will try to fix it manually, thanks so much for your great work man!
